tests/fixtures.py

```diff
@@ -366,8 +360,8 @@ def policy_rate():
 
 
 @pytest.fixture(scope='module')
-def policy_value(application_economics, policy_rate):
-    value = policy_rate * application_economics.min_operator_seconds
+def policy_value(policy_rate):
+    value = policy_rate * MIN_OPERATOR_SECONDS
```

Comment:
What does this constant represent?

Reply:
It's the minimum amount of time before an operator can be changed. It's the same value used in the TACoApplication contract.